File: Baekjoon/Problem_1000~1999/Problem_1094/Problem_1094.cpp
#include <cstdio>

using namespace std;

int main() {
    int X; scanf("%d", &X);

    int cnt = 0;
    int _sum = 0;
    for (int i = 6; i >= 0; i--) {
        if (_sum == X) break;

        int rest = (1 << i);

        if (_sum + rest <= X) {
            _sum += rest;
            cnt += 1;
        }
    }

    printf("%d\n", cnt);

    return 0;
}

File: Baekjoon/Problem_1000~1999/Problem_1094/Problem_1094.py
import sys

input = lambda: sys.stdin.readline().rstrip()

if __name__ == "__main__":
    X = int(input())
    
    cnt = 0
    _sum = 0
    for i in range(6, -1, -1):
        if _sum == X: break
        rest = 1 << i
        
        if _sum + rest <= X:
            _sum += rest
            cnt += 1
    
    print(cnt)